scipy/special/basic.py

```diff
@@ -2462,3 +2462,61 @@ def zeta(x, q=None, out=None):
         q = 1
     return _zeta(x, q, out)
 
+
+def owens_t(h, a):
+    r"""
+    Owen's T function.
+
+    Parameters
+    ----------
+    h: int or float
+        Input value.
+    a: int or float
+        Input value.
+        If ``a == 0``, the return value is 0.
+
+    Returns
+    -------
+    t: float
+        Probability of the event (X > h and 0 < Y < a * X),
+        where X and Y are independent standard normal random variables.
+
+    References
+    ----------
+    .. [1] Mike Patefield and David Tandy. "Fast and accurate calculation
+           of Owen's T-function", 2000.
+           https://www.jstatsoft.org/article/view/v005i05/t.pdf
```

Comment:
No link please, but a complete citation:

   M. Patefield and D. Tandy, "Fast and accurate calculation of Owen’st function",
   Statistical Software vol. 5, pp. 1-25, 2000.